### PR TITLE
Setup cob4-9

### PR DESCRIPTION
**Pull request by NadiaHG**
This PR requires: https://github.com/ipa320/cob_supported_robots/pull/4 and https://github.com/ipa320/cob_calibration_data/pull/130

### CODE REVIEW

File: cob_bringup/robots/cob4-9.xml [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+			<arg name="sim" value="$(arg sim)"/>
+		</include>
+		<include file="$(find cob_bringup)/tools/robot_state_publisher.launch">
+			<arg name="robot" value="$(arg robot)"/>
+		</include>
+		<include file="$(find cob_script_server)/launch/script_server.launch"/>
+
+		<!-- simulation only -->
+		<include if="$(arg sim)" file="$(find cob_bringup)/tools/fake_diagnostics.launch">
+			<arg name="fake_diagnostics" value="'base_laser_front, base_laser_right, base_laser_left, -b1, -t1, -t2, -t3, -s1, -h1, joy, safety'"/>
```
Comment (fmessmer) [inline cob_bringup/robots/cob4-9.xml]: remove fake_diagnostics for `-t1, -t2, -t3, -s1, -h1`

File: cob_default_robot_config/robots/cob4-9/base_configurations.yaml [hunk 2]
```diff
@@ -0,0 +1,2 @@
+service_ns: /base/driver
```
Comment (fmessmer) [inline cob_default_robot_config/robots/cob4-9/base_configurations.yaml]: please obey new structure, i.e. move yaml files to `script_server` subfolder

File: cob_default_robot_config/robots/cob4-9/command_gui_buttons.yaml [hunk 3]
```diff
@@ -0,0 +1,12 @@
+#group:
```
Comment (fmessmer) [inline cob_default_robot_config/robots/cob4-9/command_gui_buttons.yaml]: please obey new structure, i.e. move yaml files to `script_server` subfolder

File: cob_default_robot_config/robots/cob4-9/upload_param_cob4-9.launch [hunk 4]
```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0"?>
+<launch>
+
+	<!-- send ROBOT parameters to parameter server -->
+	<rosparam command="load" ns="/script_server/base" file="$(find cob_default_robot_config)/robots/cob4-9/base_configurations.yaml"/>
```
Comment (fmessmer) [inline cob_default_robot_config/robots/cob4-9/upload_param_cob4-9.launch]: please obey new structure, i.e. adjust to yaml files being in `script_server` subfolder

File: cob_hardware_config/robots/cob4-9/config/scan_unifier_config.yaml [hunk 5]
```diff
@@ -0,0 +1,4 @@
+#Scan_unifier_config-parameters
+
+input_scans: ["base_laser_front/scan","base_laser_left/scan","base_laser_right/scan"]
+loop_rate: 100.0
```
Comment (fmessmer) [inline cob_hardware_config/robots/cob4-9/config/scan_unifier_config.yaml]: `loop_rate` is no longer a parameter needed by the `scan_unifier` (https://github.com/ipa320/cob_driver/pull/324/files#diff-4b8c9c7e0bb817a56030c63677f60462L96)
we should remove it for all robots

File: cob_hardware_config/robots/cob4-9/urdf/properties.urdf.xacro [hunk 6]
```diff
@@ -0,0 +1,121 @@
...
+	<!-- laser top mount positions | relative to base_link -->
+	<xacro:property name="laser_right_x" value="${def_laser_right_x+offset_laser_right_x}"/>
+	<xacro:property name="laser_right_y" value="${def_laser_right_y+offset_laser_right_y}"/>
+	<xacro:property name="laser_right_z" value="${def_laser_right_z+offset_laser_right_z}"/>
+	<xacro:property name="laser_right_roll" value="${def_laser_right_roll+offset_laser_right_roll}"/>
+	<xacro:property name="laser_right_pitch" value="${def_laser_right_pitch+offset_laser_right_pitch}"/>
+	<xacro:property name="laser_right_yaw" value="${def_laser_right_yaw+offset_laser_right_yaw}"/>
+
+	<!-- *********************** -->
+	<!-- upper robot calibration -->
```
Comment (fmessmer) [inline cob_hardware_config/robots/cob4-9/urdf/properties.urdf.xacro]: remove all `property` tags below...

**Review by fmessmer — CHANGES_REQUESTED** [review]

**Review by fmessmer — APPROVED** [review]

**Comment by fmessmer** [comment]
I rebased and finalized this PR
@ipa-fmw please test again on cob4-9 hardware according to https://github.com/ipa320/cob4/issues/297#issuecomment-297689933

**Comment by NadiaHG** [comment]
thanks @ipa-fxm your changes look good to me, tested? can be merged?

**Review by fmessmer — APPROVED** [review]
successfully tested on `cob4-9` again...works...

**Pull request merged**